src/Microsoft.Extensions.Configuration/ConfigurationReloadToken.cs

```diff
@@ -7,16 +7,35 @@
 
 namespace Microsoft.Extensions.Configuration
 {
+    /// <summary>
+    /// Implements <see cref="IChangeToken"/>
+    /// </summary>
     public class ConfigurationReloadToken : IChangeToken
     {
         private CancellationTokenSource _cts = new CancellationTokenSource();
 
+        /// <summary>
+        /// Indicates if this token will pro-actively raise callbacks. Callbacks are still guaranteed to fire, eventually.
+        /// </summary>
         public bool ActiveChangeCallbacks => true;
 
+        /// <summary>
+        /// Gets a value that indicates if a change has occured.
+        /// </summary>
         public bool HasChanged => _cts.IsCancellationRequested;
 
+        /// <summary>
+        /// Registers for a callback that will be invoked when the entry has changed. Microsoft.Extensions.Primitives.IChangeToken.HasChanged
+        /// MUST be set before the callback is invoked.
+        /// </summary>
+        /// <param name="callback">The System.Action`1 to invoke.</param>
+        /// <param name="state">State to be passed into the callback.</param>
+        /// <returns></returns>
         public IDisposable RegisterChangeCallback(Action<object> callback, object state) => _cts.Token.Register(callback, state);
 
+        /// <summary>
+        /// Used to fires the change token when a reload occurs.
```

Comment:
At least change to `fire`, though perhaps there's a better word for this to begin with. The docs never use the word `fire` for events. For actual .NET `event`s we use the verb `raise`, but I'm not sure how well that works here.